### PR TITLE
Add terraform import option

### PR DESCRIPTION
**Pull request by ChrisRamsayITV**

### CODE REVIEW

File: lib/dome/terraform.rb [hunk 1]
```diff
@@ -229,6 +229,17 @@ def statecmd(arguments)
       execute_command(command, failure_message)
     end
 
+    def import(arguments)
+      params = ''
+      case level
+      when 'environment', 'services'
+        params = " -var-file=params/env.tfvars"
+      end 
+      command         = "terraform import#{params} #{arguments}"
```
Comment (xiii) [inline lib/dome/terraform.rb:238]: space missing @ChrisRamsayITV ?

**Review by anne23 — APPROVED** [review]
oh, nice! I always assumed dome just deliberately didn't do it for some good reason, didn't realise it'd be so simple to add

File: lib/dome/terraform.rb (hunk 1 above)
Comment (ChrisRamsayITV) [reply lib/dome/terraform.rb:238]: no - it's empty if not set - the space is at the start of the string

**Comment by ChrisRamsayITV** [comment]
> oh, nice! I always assumed dome just deliberately didn't do it for some good reason, didn't realise it'd be so simple to add

same - it just got to the point where I figured it would save me time in the long run and turned out to be a 5 minute fix

**Pull request merged**